GETTING-STARTED.md

```diff
@@ -31,8 +41,13 @@ $ pip install django-cloud-deploy
 
 ## Creating a New Project
 
+At the end of this step, you will have a skeleton Django project that is
+deployed on
+[Kubernetes Engine](https://cloud.google.com/kubernetes-engine/). You can
```

Comment:
The last sentence seems unnecessary.

Reply:
I like it. I changed the wording a bit but done.